sale_order_revision/view/sale_order.xml

```diff
@@ -1,28 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
-        <record id="sale_order_form" model="ir.ui.view">
-            <field name="name">sale.order.form</field>
-            <field name="model">sale.order</field>
-            <field name="inherit_id" ref="sale.view_order_form"></field>
-            <field name="arch" type="xml">
-                <notebook position="inside">
-                    <page string="Revisions">
-                        <field name="old_revision_ids" >
-                            <tree>
-                                <field name='name'/>
-                                <field name='create_date' string="Superseeded on"/>
-                                <field name='create_uid' string="Superseeded by"/>
-                                <field name='state' invisible='1'/>
-                            </tree>
-                        </field>
-                        <group attrs="{'invisible': [('active', '=', True)]}">
-                            <field name="current_revision_id"/>
-                            <field name="active" invisible="1"/>
-                        </group>
-                    </page>
-                </notebook>
-            </field>
-        </record>
-    </data>
-</openerp>
+<odoo>
+    <record id="sale_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"></field>
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Revisions">
+                    <field name="old_revision_ids" >
+                        <tree>
+                            <field name='name'/>
+                            <field name='create_date' string="Superseeded on"/>
+                            <field name='create_uid' string="Superseeded by"/>
+                            <field name='state' invisible='1'/>
+                        </tree>
+                    </field>
+                    <group attrs="{'invisible': [('active', '=', True)]}">
+                        <field name="current_revision_id"/>
+                        <field name="active" invisible="1"/>
+                    </group>
+                </page>
+            </notebook>
+            <xpath expr="//header/button[@name='action_draft']" postion='after'>
+                <button name="copy_quotation" states="cancel" string="New Copy of Quotation" type="object"/>
```

Comment:
I think it would be better to call this button 'New revision of quotation' instead of copy, so it's easier to know we don't use the standard duplicate.